ext/src/nassau.rs

```diff
@@ -440,6 +462,12 @@ impl Resolution {
             assert!(dx.is_zero(), "dx non-zero at t = {t}, s = {s}");
         }
         self.differential(s).add_generators_from_rows(t, xs);
+        let stop_the_clock = timer.elapsed();
+        eprintln!(
+            "[{:>6}.{:>06} s] Computed bidegree ({s}, {t}) with {subalgebra}",
+            stop_the_clock.as_secs(),
+            stop_the_clock.subsec_micros()
+        );
```

Comment:
Don't have to change it in this PR, but I usually like to print the data in csv format for further postpocessing, e.g. generating graphs